defaults/main.yml

```diff
@@ -5,7 +5,9 @@ drupal_deploy: false
 drupal_deploy_repo: ""
 drupal_deploy_version: master
 drupal_deploy_update: true
+drupal_deploy_depth: 1
```

Comment:
This is an additional change which may (often) not be desirable as a default. I would put it as `""`; and then see other comment in `deploy.yml`.